README.md

```diff
@@ -1,10 +1,18 @@
-# About Eclipse Che Dashboard
+## About Eclipse Che
 
-This is the first step to implement a new Eclipse Che Dashboard.
+Eclipse Che is a next generation Eclipse IDE. This repository is licensed under the Eclipse Public License 2.0. Visit [Eclipse Che's Web site](https://eclipse.org/che/) for feature information or the main [Che assembly repository](https://github.com/eclipse/che) for a description of all participating repositories.
+
+# Eclipse Che Dashboard
 
 ## Requirements
 
 - Node.js `v10.x.x` and later.
+- yarn `v1.13.0` or higher. https://yarnpkg.com/lang/en/docs/install/
```

Comment:
I think it should be ```yarn `v1.20.0` or higher```.

[```"yarn": ">=1.20.0"```](https://github.com/che-incubator/che-dashboard-next/blob/master/package.json#L20)